Neos.Neos/Documentation/Operations/CommandLine.rst

```diff
@@ -81,31 +81,31 @@ create and delete workspaces as well as publish and discard changes.
 One notable difference is that rebasing a workspace is possible from the command line *even if it contains
 unpublished changes*.
 
-=======================================  ========================================
+=======================================  ===================================================
 Command                                  Description
-=======================================  ========================================
+=======================================  ===================================================
 workspace:publish                        Publish changes of a workspace
 workspace:discard                        Discard changes in workspace
 workspace:create                         Create a new workspace
 workspace:delete                         Deletes a workspace
 workspace:rebase                         Rebase a workspace
 workspace:list                           Display a list of existing workspaces
-=======================================  ========================================
+=======================================  ===================================================
 
 Site Management
 ===============
 
-=======================================  ========================================
+=======================================  ===================================================
 Command                                  Description
-=======================================  ========================================
+=======================================  ===================================================
 domain:add                               Add a domain record
 domain:list                              Display a list of available domain
                                          records
-domain:delete                            Delete a domain record
-domain:activate                          Activate a domain record
-domain:deactivate                        Deactivate a domain record
+domain:delete                            Delete a domain record (with globbing)
```

Comment:
I am not sure if delete is a good idea 🤣

Reply:
Why not? If you pass `./flow domain:delete "*"` you will get what you asked for.

Reply:
@kitsunet To clarify a bit, the use case i have in mind for that is to remove all domain configurations after cloning a db to a test or staging environment and add the local ones afterwards. That way the logic does not have to be adjusted if more domains are added to the live-site.